crates/sui-core/src/authority/authority_store.rs

```diff
@@ -384,15 +384,15 @@ impl AuthorityStore {
     /// Returns None otherwise
     pub async fn execution_lock_for_certificate(
         &self,
-        certificate: &CertifiedTransaction,
+        certificate: &VerifiedExecutableTransaction,
```

Comment:
we should probably rename this method and the parameter since it may not be a certificate anymore